lib/datadog/search-syntax/src/vrl.rs

```diff
@@ -0,0 +1,187 @@
+use super::{
```

Comment:
I would still want to share some of these methods, since I'm using them in my lib as well, but I'm ok if we merge it and move to a dedicated lib later.